phase1/gce/configure-vm.sh

```diff
@@ -20,6 +20,7 @@ KUBEADM_ENABLE_CLOUD_PROVIDER=$(get_metadata "k8s-kubeadm-enable-cloud-provider"
 KUBEADM_ADVERTISE_ADDRESSES=$(get_metadata "k8s-kubeadm-advertise-addresses")
 KUBEADM_CNI_PLUGIN=$(get_metadata "k8s-kubeadm-cni-plugin")
 KUBEADM_MASTER_IP=$(get_metadata "k8s-kubeadm-master-ip")
+KUBEPROXY_MODE=$(get_metadata "kube-proxy-mode")
```

Comment:
Nitpick, but for consistency can you name this `k8s-kube-proxy-mode` or something similar with a `k8s-` prefix? It just makes it easier to find all of the metadata related to Kubernetes. This won't affect user configuration, so I don't mind if you want to merge first for testing and then rename this in a future PR.